js/core.js

```diff
@@ -168,15 +168,24 @@
         /**
          * should be used to log information on the server to different log locations
          * @param {Object} headers - the headers to be logged, should include action
-         * @param {String} location - the path for logging (terminal_error || button_action)
+         * @param {String} location - the path for logging (terminal_error || client_action)
          **/
-        this.logHeaders = function (actionHeader, location) {
+        this.logHeaders = function (contextHeaderParams, location) {
```

Comment:
Do we need to pass the `location`?  Currently, the `terminal_error` is specified only in ermrestjs and chaise has no knowledge of that. Why chaise should send the location in this case? Are we going to use multiple locations for our logs?